Directory.Packages.props

```diff
@@ -38,8 +38,8 @@
     </ItemGroup>
     <!-- Semantic Kernel -->
     <ItemGroup>
-        <PackageVersion Include="Microsoft.SemanticKernel.Abstractions" Version="1.0.0-rc3"/>
-        <PackageVersion Include="Microsoft.SemanticKernel.Connectors.AI.OpenAI" Version="1.0.0-rc3"/>
+        <PackageVersion Include="Microsoft.SemanticKernel.Abstractions" Version="1.0.0-rc4"/>
+        <PackageVersion Include="Microsoft.SemanticKernel.Connectors.AI.OpenAI" Version="1.0.0-rc4"/>
```

Comment:
The Microsoft.SemanticKernel.Connectors.AI.OpenAI package was deprecated on RC4 the replacement is Microsoft.SemanticKernel.Connectors.OpenAI